src/App.jsx

```diff
@@ -12,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<ArticlesList/>}/>
 				<Route path="/article/:id" element={<Article/>} />
+				<Route path="/search/:topic" element={<SearchPage/>} />
```

Comment:
Nice use of parametric endpoints here so that the user can share the page organised by topic ⭐